be/src/runtime/load_channel.h

```diff
@@ -88,9 +92,20 @@ class LoadChannel {
     Span get_span() { return _span; }
 
 private:
+    void _add_chunk(vectorized::Chunk* chunk, const PTabletWriterAddChunkRequest& request,
+                    PTabletWriterAddBatchResult* response);
+    Status _build_chunk_meta(const ChunkPB& pb_chunk);
+    Status _deserialize_chunk(const ChunkPB& pchunk, vectorized::Chunk& chunk, faststring* uncompressed_buffer);
+
     LoadChannelMgr* _load_mgr;
     UniqueId _load_id;
     int64_t _timeout_s;
+    std::atomic<bool> _has_chunk_meta;
+    mutable bthread::Mutex _chunk_meta_lock;
+    serde::ProtobufChunkMeta _chunk_meta;
```

Comment:
All `TabletsChannel` in the same `LocdChannel` have the same `ProtobufChunkMeta`?

Reply:
yes